src/targets/gpu/include/migraphx/gpu/hip.hpp

```diff
@@ -103,6 +103,14 @@ struct hip_copy_to_gpu
 
 struct hip_copy_from_gpu
 {
+    bool sync_copy = false;
```

Comment:
This should be removed. A `hip_sync_device` should be inserted after the copy if it needs to be synced. We might need to insert an identity operator so scheduling won't change the order.

Reply:
changed